views/pod.slim

```diff
@@ -38,6 +38,23 @@ ruby:
     eos
   end
 
+  stats = Hash[@stats.to_h.map do |k, v|
```

Comment:
I felt it was a good fit for the view because it is inherently a "view" thing. Since it will be different for a view in `en-US` and `sv-SE` theoretically. Where would you suggest putting it instead?